spec/Overview.html

```diff
@@ -2286,6 +2465,31 @@ <h4>The {{SubtleCrypto/exportKey()}} method</h4>
                   internal slot of |key| using |key| and |format|.
                 </p>
               </li>
+              <li>
+                <p>
+                  [= Queue a global task =] on the [= crypto task source =],
+                  given |realm|'s global object, to perform the remaining steps.
+                </p>
+              </li>
+              <li>
+                <dl class="switch">
+                  <dt>
+                    If |format| is equal to the strings {{KeyFormat/"raw"}},
+                    {{KeyFormat/"pkcs8"}}, or {{KeyFormat/"spki"}}:
```

Comment:
Infra keeps the quotation marks outside of the code markup. `"{{KeyFormat/raw}}"` works for that.

Reply:
OK, yeah. I'll make a separate PR for that as this is used all over the place.

Reply:
Seems the spec is in contradiction to the last paragraph of https://w3c.github.io/webcrypto/#conformance

Reply:
I think that text was more relevant before https://github.com/w3c/webcrypto/pull/309, in which I put many quotes outside `<code>`.

... I was gonna say ".. but apparently not all of them", but it seems the markup works fine as is, as well: https://w3c.github.io/webcrypto/#SubtleCrypto-method-importKey (which has similar markup as this one).